JM/CF/Scripts/1_Core/CommunityFramework/CF_LogLevel.c

```diff
@@ -0,0 +1,11 @@
+enum CF_LogLevel
+{
+	ERROR	= 0x000001,
+	WARNING	= 0x000010,
+	INFO 	= 0x000100,
+	DEBUG	= 0x001000,
+	TRACE	= 0x010000,
+
+	NONE	= 0,
+	ALL		= 0x011111,
```

Comment:
Why is there an ALL option. Log levels should be exclusive and not a bitmask to be combined.